package.json

```diff
@@ -27,7 +27,7 @@
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
-    "tailwindcss": "^1.8.12"
+    "tailwindcss": "^1.9.5"
```

Comment:
just make sure to include the PR description that you bumped up dependencies. Also make sure to post on slack that you're doing so

Reply:
Thanks for point it out. I was not aware of this. I was messing with tailwind, tried to customize some style, seems like I updated the version...